packages/os/prairiedog-toml

```diff
@@ -0,0 +1,22 @@
+[required-extensions]
+boot = "v1"
++++
+{{#if settings.boot}}
+{{#if settings.boot.reboot-to-reconcile}}
+reboot-to-reconcile = {{settings.boot.reboot-to-reconcile}}
+{{/if}}
+
+{{#if settings.boot.kernel}}
+[kernel]
+{{#each settings.boot.kernel}}
+"{{@key}}" = [ {{#each this}}"{{this}}",{{/each}} ]
+{{/each}}
```

Comment:
Does this render correctly when the values contain quotes or spaces?
```
$ apiclient apply <<EOF
settings.boot.kernel-parameters]
param = ["\"with quotes\""]
EOF
```

Reply:
Isn't this more of a question for schnauzer than necessarily the template file, or is there a helper method I need to be using for the above?

Reply:
Yeah i need to change {{this}} to {{{this}}} otherwise it html escapes